kpops/components/streams_bootstrap/streams/streams_app.py

```diff
@@ -13,7 +15,10 @@ class StreamsApp(KafkaApp):
     StreamsApp component that configures a streams bootstrap app
     """
 
-    _type = "streams-app"
+    type: ClassVar[str] = "streams-app"
+    schema_type: Literal["streams-app"] = Field(
+        default="streams-app", alias="schema_type", exclude=True
```

Comment:
Do we need the `alias` property here? Since the name of the field is exactly the same as the `alias`

Reply:
yeah, unfortunately, otherwise it automatically gets a camelcase alias. Perhaps we could exclude this field from the camelcase generator somehow?

Reply:
Yeah, if that is not complicated 😄 because this is, at first glimpse, a bit confusing. Otherwise, it is fine.